pywbem/_cim_operations.py

```diff
@@ -634,12 +647,12 @@ def __init__(self, url, creds=None, default_namespace=None,
             and should be avoided!
 
           timeout (:term:`number`):
-            Timeout in seconds, for requests sent to the server.
+            Timeout in seconds, for requests sent to the server or listener.
 
             *New in pywbem 0.8.*
 
-            If the server did not respond within the timeout duration, the
-            socket for the connection will be closed, causing a
+            If the server or listener did not respond within the timeout
```

Comment:
change dit not respond to does not respond.

Reply:
DONE